docs/examples/resnet50_trt/README.md

```diff
@@ -0,0 +1,247 @@
+
+
+#### Dependencies
+
+* Export ONNX and build TensorRT
+  * nvcr.io/nvidia/pytorch:20.12-py3
+* Triton Inference Server for DALI backend
+  * nvcr.io/nvidia/tritonserver:20.12-py3
+  * https://github.com/triton-inference-server/dali_backend
+
+* Client
+  * nvcr.io/nvidia/tritonserver:20.10-py3-clients
+
+#### Setting up the ONNX-TensorRT and DALI backend ENV
+
+```
+$git clone https://gitlab-master.nvidia.com/ghong/triton_dali
+$cd triton_dali
+$git clone https://github.com/triton-inference-server/dali_backend
+$cd dali_backend
+$docker build -t triton_dali_backend -f Dockerfile .
+$cd ..
+```
+
+####  Build TensorRT via ONNX
+
+##### 1.  Converting PyTorch Model to ONNX-model 
+
+Run `onnx_exporter.py` for conversion using PyTorch model to ONNX model. In this case, ResNet50 model is converted to ONNX format. `width` and `height` dims are fixed at 224 but dynamic axes arguments for dynamic batch is used. 
+
+```
+$docker run -it --gpus=all -v $(pwd):/workspace nvcr.io/nvidia/pytorch:20.12-py3 bash
+$python onnx_exporter.py --save model.onnx
+```
+
+##### 2. Building ONNX-model to TensorRT engine
+
+Set the arguments for enabling fp16 precision `--fp16` and for dynamic shapes using a profile `--minShapes`, `--optShapes`, and `maxShapes` with `--explicitBatch`
+
+```python
+$trtexec --onnx=model.onnx --saveEngine=./model_repository/resnet50_trt/1/model.plan --explicitBatch --minShapes=input:1x3x224x224 --optShapes=input:1x3x224x224 --maxShapes=input:256x3x224x224 --fp16
+```
+
+##### 3. Serialize DALI pipeline 
+
+Run `serialize_dali_pipeline.py` for generating  DALI pipeline. 
+
+```
+$python serialize_dali_pipeline.py --save ./model_repository/dali/1/model.dali
+```
+
+If had a dedicated hardware decoder, the hardware is available. Set the preprocessing pipeline  `resize` and `normalize` using `dali.pipeline.Pipeline` and serialize pipeline using `Pipeline.serialize`
+
+````python
+pipe = dali.pipeline.Pipeline(batch_size=256, num_threads=4, device_id=0)
+with pipe:
+    images = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
+    images = dali.fn.image_decoder(images, device="mixed", output_type=types.RGB)
+    images = dali.fn.resize(images, resize_x=224, resize_y=224)
+    images = dali.fn.crop_mirror_normalize(images,
+                                           dtype=types.FLOAT,
+                                           output_layout="CHW",
+                                           crop=(224, 224),
+                                           mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
+                                           std=[0.229 * 255, 0.224 * 255, 0.225 * 255])
+
+
+    pipe.set_outputs(images)
+    pipe.serialize(filename=args.save)
+````
+
+
+
+#### Run Triton Inference Server
+
+![](./images/ensemble.PNG)
+
+```bash
+model_repository
+├── dali
+│   ├── 1
+│   │   └── model.dali
+│   └── config.pbtxt
+├── ensemble_dali_resnet50
+│   ├── 1
```

Comment:
This directory is also lacking

Reply:
update README.md

Reply:
Add commands for creating directories. [README.md](https://github.com/triton-inference-server/dali_backend/pull/35/commits/e64638fc5015b57260cd114e6723c81fdb493744#diff-3272a2f918db29f27589316303368b47f9187f0910d33ec7fb7d6f18868ceea2R42)